addon/-base-url.js

```diff
@@ -36,6 +36,10 @@ class BaseURL {
     this._setPart('host', value);
   }
 
+  get origin() {
```

Comment:
Should I add empty `set` method as well?

Reply:
It's a readonly attribute in spec, I guess the set method should throw.